pkg/cmdutil/json_flags.go

```diff
@@ -121,6 +173,18 @@ type exportFormat struct {
 	template string
 }
 
+// NewJSONExporter returns an Exporter to emit JSON.
+func NewJSONExporter(opts ...JSONExporterOption) Exporter {
+	e := &exportFormat{}
+	for _, opt := range opts {
+		opt(e)
+	}
+	return e
+}
+
+// JSONExporterOption customizes a JSON Exporter for NewJSONExporter.
+type JSONExporterOption func(e *exportFormat)
```

Comment:
I figured they might coeme in handy later and since it's technically a public API, figured I'd get the signature right now (helping others avoiding breaking changes in Azure APIs is part of my day job). But, it's an EXE, so maybe this isn't important. I can remove it if you'd rather.

Reply:
Removed.